h/security/policy/_identity_base.py

```diff
@@ -45,9 +45,3 @@ def authenticated_userid(self, request):
             return identity.user.userid
 
         return None
-
-    def remember(self, _request, _userid, **_kwargs):
-        return []
-
-    def forget(self, _request):
-        return []
```

Comment:
API sub-policies shouldn't have `remember()` and `forget()` methods since those methods make no sense for stateless API endpoints and the top-level `APIPolicy` will never call them, so I don't want API sub-policies that inherit from `IdentityBasedPolicy` to inherit these methods.

These `IdentityBasedPolicy.remember()` and `IdentityBasedPolicy.forget()` methods also have nothing to do with `IdentityBasedPolicy`'s purpose of helping to make policies whose `authenticated_userid()` and `permits()` methods are based on the return value of their `identity()` method.

There's actually only one class that inherits these methods, doesn't override them, and actually needs/wants to have them.